playwright.config.ts

```diff
@@ -0,0 +1,155 @@
+import { PlaywrightCoverageOptions } from "@bgotink/playwright-coverage";
+import { defineConfig, devices, ReporterDescription } from "@playwright/test";
+import path from "path";
+
+const collectCoverage = !!process.env.CI;
+const coverageReporter: ReporterDescription = [
+  "@bgotink/playwright-coverage",
+  /** @type {import('@bgotink/playwright-coverage').CoverageReporterOptions} */ {
+    /* Path to the root files should be resolved from */
+    sourceRoot: __dirname,
+    /* The coverage is reported with a prefix. This is probably because we are
+       running in an iframe so the coverage of CODAP and the plugin is differentiated
+       by the two prefixes. */
+    exclude: [
+      /* Ignore all coverage of the CODAP application itself. */
+      "codap3/src/**", "codap3/webpack/**",
+      /* This repository has a copy of many of the CODAP files which it uses like a
+         library to handle its dataset and add the legend component. These files are
+         identical to the files in CODAP and should be tested there not here. So we
+         ignore them here. */
+      "codap-plugin-starter-project/src/codap/**"
```

Comment:
This might be useful to have in general for codap plugins, but the starter project doesn't actually include any of codap, does it? If I'm right about that, I think it would at least make sense to change the comments here to explain that plugins frequently copy over code from codap, and how to do that, but that it hasn't been done for the starter plugin yet.

Reply:
Yes, good catch, I thought I had deleted that.

Reply:
I would note that the first part on line 16 is still needed. Because the tests are running CODAP with the plugin iframed, the coverage includes both the codap application and the plugin. So the `codap3/` prefix is there to ignore those CODAP application files.